app/javascript/topNavigation/utilities.js

```diff
@@ -63,17 +63,6 @@ export function initializeTouchDevice(memberTopMenu, menuNavButton) {
         menuNavButton.addEventListener('click', (_event) => {
           toggleHeaderMenu(memberTopMenu, menuNavButton);
         });
-
-        document.addEventListener('click', (_event) => {
-          // if clicking outside of the menu or on a menu item, close it
-          if (document.activeElement.id !== 'member-menu-button') {
```

Comment:
@katiedavis in the mean time I played with this for a bit and adding this extra check (for the click event itself) fixed the problem for me (without rolling back):

```
if (document.activeElement.id !== 'member-menu-button' && event.target.id !== 'nav-profile-image') {
```

Not sure if you want to send in a follow up PR or add this as a fix instead of rolling back... Either way totally okay for me! 😃